### DISCUSSION TITLE
Git for Windows v2.45.2.windows.1

### DISCUSSION THREAD

**Discussion by dscho** [git-for-windows/git#5035] [Announcements]
Changes since Git for Windows v2.45.1 (May 14th 2024)

Git for Windows for Windows v2.45 is the last version to support for Windows 7 and for Windows 8, see [MSYS2's corresponding deprecation announcement](https://www.msys2.org/docs/windows_support/) (Git for Windows relies on MSYS2 for components such as Bash and Perl).

Please also note that the 32-bit variant of Git for Windows is deprecated; Its last official release [is planned for 2025](https://gitforwindows.org/32-bit.html).

### New Features

* Comes with [Git v2.45.2](https://github.com/git/git/blob/v2.45.2/Documentation/RelNotes/2.45.2.txt).
* Comes with [Tig v2.5.10](https://github.com/jonas/tig/releases/tag/tig-2.5.10).
* Comes with [cURL v8.8.0](https://github.com/curl/curl/releases/tag/curl-8_8_0).

### Bug Fixes

* When Git for Windows v2.44.0 introduced the ability [to use native Win32 Console ANSI sequence processing](https://github.com/git-for-windows/git/pull/4700), an inadvertent fallout was that in this instance, [non-ASCII characters were no longer printed correctly unless the current code page was set to 65001](https://github.com/git-for-windows/git/issues/4851). This bug [has been fixed](https://github.com/git-for-windows/git/pull/4968).

Filename | SHA-256
-------- | -------
Git-2.45.2-64-bit.exe | ce022a6a19e58bbbd4823f51cf798b006b4a683b93b0616a7bb5beeee901da98
Git-2.45.2-32-bit.exe | b8e6db9004c602d9dc375196244484399eb3e33ad71ba3c71455dde4487b4ce6
PortableGit-2.45.2-64-bit.7z.exe | 851a15074dea6b272785b2a2a4697a72970256de2afe7b8e4a9c5e168c27ccdd
PortableGit-2.45.2-32-bit.7z.exe | cfea9e414567d0c59c75ccc5a0e58feeef4dcfc0ea8bfd76efb5e4e22813f5d0
MinGit-2.45.2-64-bit.zip | 7ed2a3ce5bbbf8eea976488de5416894ca3e6a0347cee195a7d768ac146d5290
MinGit-2.45.2-32-bit.zip | 0c0251d02c322d208fb59ba81e7d72db621cd53a3f93b105677dab4dc565199e
MinGit-2.45.2-busybox-64-bit.zip | e841b7aefa8d166fb9dd91afcfb42f163ae6f8d9bd9061da9856084af214a011
MinGit-2.45.2-busybox-32-bit.zip | 9e8ede5629d928f943909f0f3225b27716bdaa95018a3c6cf6276fbc99ee19da
Git-2.45.2-64-bit.tar.bz2 | ef73e007a44a15e8c21fed165faea8707a28703c21c326ac609d706225ba1662
Git-2.45.2-32-bit.tar.bz2 | 451e476b925842546fe47429efcfa91185727da2c2477413556e7827daeab847

<hr /><em>This discussion was created from the release <a href='https://github.com/git-for-windows/git/releases/tag/v2.45.2.windows.1'>Git for Windows v2.45.2.windows.1</a>.</em>

**Comment by Gideon-gfx** [comment]
it's not downloading

**Reply by dscho** [reply]
Hmm. It works for me when I click on the <kbd>Download</kbd> button on https://gitforwindows.org/. Maybe you can share any details about any symptoms, e.g. via your browser's Developer Tools?

**Comment by Gideon-gfx** [comment]
your files isn't downloading
